portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/searchtuning/SearchExperiences.testcase

```diff
@@ -0,0 +1,601 @@
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
+
+		SearchPortlets.searchEmbedded(searchTerm = "location");
+
+		SearchResultPortlet.viewSearchResultsSpecificOrder(resultsList = "Closest Location,Closer Location,Close Location");
+	}
+
+	test CreateBoostAssetType {
+		property test.name.skip.portal.instance = "SearchExperiences#CreateBoostAssetType";
+
+		SearchExperiences.openSearchExperiences(siteURLKey = "guest");
+
+		SearchExperiences.addBlueprint(name = "My Blueprint");
+
+		var portalURL = PropsUtil.get("portal.url");
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
+
+		SearchPortlets.addWidgets(searchPortletList = "Blueprints Options");
+
+		SearchExperiences.selectBlueprintFromOptionsWidget(name = "My Blueprint");
+
+		JSONWebcontent.addWebContent(
+			content = "Liferay",
+			groupName = "Guest",
+			title = "Liferay Web Content");
+
+		JSONBlog.addEntry(
+			entryContent = "Liferay Liferay",
+			entryTitle = "Liferay Blogs Entry");
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
+
+		SearchPortlets.searchEmbedded(searchTerm = "liferay");
+
+		SearchResultPortlet.viewSearchResultsSpecificOrder(resultsList = "Liferay Blogs Entry,Liferay Web Content");
+
+		SearchExperiences.openSearchExperiences(siteURLKey = "guest");
+
+		SearchExperiences.editBlueprint(name = "My Blueprint");
+
+		SearchExperiences.addQueryElements(
+			assetType = "Web Content Article",
+			fragment = "Boost Asset Type",
+			numberBoost = "1000");
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
+
+		SearchPortlets.searchEmbedded(searchTerm = "liferay");
+
+		SearchResultPortlet.viewSearchResultsSpecificOrder(resultsList = "Liferay Web Content,Liferay Blogs Entry");
+	}
+
+	@priority = "4"
+	test HideWebContentWithCustomElement {
+		property test.name.skip.portal.instance = "SearchExperiences#HideWebContentWithCustomElement";
+
+		SearchExperiences.openSearchExperiences(siteURLKey = "guest");
+
+		SearchExperiences.addBlueprint(name = "Hide Test and Experimental Content");
+
+		var portalURL = PropsUtil.get("portal.url");
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
+
+		SearchPortlets.addWidgets(searchPortletList = "Blueprints Options");
+
+		SearchExperiences.selectBlueprintFromOptionsWidget(name = "Hide Test and Experimental Content");
+
+		JSONWebcontent.addWebContent(
+			content = "",
+			groupName = "Guest",
+			title = "Content");
+
+		JSONWebcontent.addWebContent(
+			assetTagNames = "test",
+			content = "",
+			groupName = "Guest",
+			title = "Test Content");
+
+		JSONWebcontent.addWebContent(
+			assetTagNames = "experimental",
+			content = "",
+			groupName = "Guest",
+			title = "Experimental Content");
+
+		JSONBlog.addEntry(
+			entryContent = "Liferay Liferay",
+			entryTitle = "Liferay Blogs Entry");
+
+		Navigator.openSpecificURL(url = "${portalURL}/web/guest/search");
+
+		SearchPortlets.searchEmbedded(searchTerm = "Content");
```

Comment:
We can get rid of this part.

These test steps follow the original story ticket. The format shows that the results with and without the blueprint are different, but the latter is redundant since we already have a tests that asserts that results show up in a specific order.